### PR TITLE
Add application membership and invite

### PR DESCRIPTION
**Pull request by nghiant2710**
Connects-to: https://github.com/balena-io/balena-sdk/pull/1050, https://github.com/balena-io/balena-sdk/pull/894
Change-type: minor
Signed-off-by: Trong Nghia Nguyen <nghiant2710@gmail.com>

### CODE REVIEW

File: balena/models/application.py [hunk 1]
```diff
@@ -881,3 +914,300 @@ def get_dashboard_url(self, app_id):
...
+            self.RESOURCE, 'GET', params=params,
+            endpoint=self.settings.get('pine_endpoint')
+        )['d']
+        
+    def create(self, app_id, invitee, role_name=None, message=None):
+        """
+        Creates a new invite for an application.
+
+        Args:
+            app_id (str): application id.
```
Comment (thgreasi) [inline balena/models/application.py]: Shouldn't the `str` here be `int`?

File: balena/models/application.py [hunk 2]
```diff
@@ -881,3 +914,300 @@ def get_dashboard_url(self, app_id):
...
+            invitee (str): the email/balena_username of the invitee.
```
Comment (thgreasi) [inline balena/models/application.py]: That's actually something we need to change in the JS SDK as well
```suggestion
            invitee (str): the email of the invitee.
```
See: https://github.com/balena-io/balena-api/blob/a0472380577b2fd516c3d64929fb42af86830b22/src/features/invitations/hooks/application.ts#L74-L76

File: balena/models/application.py [hunk 3]
```diff
@@ -881,3 +914,300 @@ def get_dashboard_url(self, app_id):
...
+        
+        data = {
+            'username': user_name,
+            'is_member_of__application': app_id
+        }
+
+        if role_name:
+            data['application_membership_role '] = _get_role_by_name(role_name)
+
+        print(data)
```
Comment (thgreasi) [inline balena/models/application.py]: `print` :police_car:

**Review by thgreasi — COMMENTED** [review]
Feel free to self-certify after confirming whether we should be documenting the id params as str or int.

**Comment by nghiant2710** [comment]
@balena-ci I self-certify!

**Pull request merged**